packages/cactus-test-tooling/src/main/typescript/localstack/localstack.ts

```diff
@@ -0,0 +1,152 @@
+import { EventEmitter } from "events";
+
+import Docker, { Container } from "dockerode";
+
+import {
+  Logger,
+  Checks,
+  LogLevelDesc,
+  LoggerProvider,
+} from "@hyperledger/cactus-common";
+
+import { Containers } from "../common/containers";
+
+export interface ILocalStackOptions {
+  imageVersion?: string;
+  imageName?: string;
+  logLevel?: LogLevelDesc;
+}
+
+export const K_DEFAULT_LOCALSTACK_IMAGE_NAME = "localstack/localstack";
+export const K_DEFAULT_LOCALSTACK_IMAGE_VERSION = "0.12.10";
+export const K_DEFAULT_LOCALSTACK_HTTP_PORT = 4566;
+
+/**
+ * > **Do not use this image in production.**
+ *
+ * Class responsible for programmatically managing a container that is running
+ * the official LocalStack image in development mode.
+ */
+export class LocalStack {
```

Comment:
@jagpreetsinghsasan I recommend renaming this to `LocalStackContainer` to make it clear that the class itself is not the entire thing that LocalStack is but rather just a small utility/helper for managing a container that is LocalStack.
If you do the renaming please also make sure to rename the .ts file itself in the same manner.